packages/nimble-tokens/source/styledictionary/config.js

```diff
@@ -228,128 +228,115 @@ const getTokenNameFromProperty = function (prop) {
 StyleDictionary.registerTransform({
     name: 'name/dsp/kebab',
     type: 'name',
-    matcher: function (_prop) {
+    filter: function (_prop) {
         return true;
     },
-    transformer: function (prop) {
+    transform: function (prop) {
         return kebabCase(getTokenNameFromProperty(prop));
     }
 });
 
+// Workaround as name/dsp/kebab does not support prefixes
+// See: https://github.com/AdobeXD/design-system-package-dsp/issues/27
+StyleDictionary.registerTransform({
+    name: 'name/nimble/kebab',
+    type: 'name',
+    filter: function (_prop) {
+        return true;
+    },
+    transform: function (prop) {
+        return `ni-nimble-base-${kebabCase(getTokenNameFromProperty(prop))}`;
+    }
+});
```

Comment:
brought over from nimble-extensions and switched from the overrides pattern to using the functions in this class directly. The overrides thing wasn't needed since we own this file instead of adobe xd dsp plugin so can simplify.